recipes/lz4-c/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set pkg_name = "lz4-c" %}
+{% set author = "lz4" %}
+{% set name = "lz4" %}
+{% set version = "1.7.5" %}
+{% set sha256sum = "0190cacd63022ccb86f44fa5041dc6c3804407ad61550ca21c382827319e7e7e" %}
+
+package:
+  name: {{ pkg_name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/{{ author }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+  patches:
+   - patches/0001-Do-not-build-tests-binaries-on-VS-project.patch  # [win]
+
+build:
+  number: 0
+  skip: True  # [win and py2k]
+  features:
+    - vc14  # [win and py>=35]
+
+requirements:
+  build:
+    - toolchain
+    - python     # [win]
+    - vc 14      # [win and py>=35]
```

Comment:
Would switch to `py35`. `conda-smithy` renders things incorrectly if they have comparisons in the selectors on dependencies.

xref: https://github.com/conda-forge/conda-smithy/issues/257